Source/WebKit/Shared/haiku/WebCoreArgumentCodersHaiku.cpp

```diff
@@ -65,7 +65,7 @@ bool ArgumentCoder<ResourceResponse>::decodePlatformData(Decoder& decoder, Resou
         return false;
     resourceResponse.setSoupMessageFlags(static_cast<SoupMessageFlags>(soupMessageFlags));
     return true;
-    */
+    //
```

Comment:
should not be here and will break the build (I used #if 0 instead of commenting out code)